CK3toEU4/Source/EU4World/Country/Country.cpp

```diff
@@ -1100,18 +1176,23 @@ void EU4::Country::assignReforms(const std::shared_ptr<mappers::RegionMapper>& r
 	else if (details.holder->getCharacterDomain()->getLaws().count("tribal_authority_0") ||
 				details.holder->getCharacterDomain()->getLaws().count("crown_authority_0"))
 		governmentType = "aristocratic";
-	else
-		governmentType = "despotic";
 
 	// MONARCHIES
 	if (details.government == "monarchy")
 	{
 		// Iqta
-		if (muslimReligions.count(details.religion) && (governmentType == "aristocratic" || governmentType == "despotic"))
+		if (muslimReligions.count(details.religion) && governmentType == "aristocratic")
```

Comment:
However, on sets you can use .contain() instead of .count() without issues. Here and below.